cmd/sealer/cmd/cluster/apply.go

```diff
@@ -58,94 +71,239 @@ will apply the diff change of current Clusterfile and the original one.`,
 			if err != nil {
 				return err
 			}
-			//save desired clusterfile
-			if err = cf.SaveAll(); err != nil {
-				return err
-			}
 
-			cluster := cf.GetCluster()
-			infraDriver, err := infradriver.NewInfraDriver(&cluster)
-			if err != nil {
-				return err
+			if !osi.IsFileExist(common.DefaultKubeConfigFile()) {
```

Comment:
You can check if k8s exists here, more security.

Reply:
init k8s client to check ?